### ISSUE TITLE
Improve ets:new/2 error when trying to create duplicate named tables

### ISSUE THREAD

**Issue by ding-an-sich** [erlang/otp#6708]
**Is your feature request related to a problem? Please describe.**
When trying to create duplicate named tables, this is the error returned to the user (Erlang/OTP 25.0.3):

```
Erlang/OTP 25 [erts-13.0.3] [source] [64-bit] [smp:32:12] [ds:32:12:10] [async-threads:1] [jit:ns]

Eshell V13.0.3  (abort with ^G)

1> ets:new(my_table, [named_table]).
my_table
2> ets:new(my_table, [named_table]).
** exception error: bad argument
     in function  ets:new/2
        called as ets:new(my_table,[named_table])
        *** argument 2: invalid options
```

**Describe the solution you'd like**

It would be better to return an error describing what really is going on (there exists a table with the given name already) or return the named table that already exists.

**Comment by jdamanalo** [comment]
Possibly #6252 and #6338

**Comment by rickard-green** [comment]
Closing this as it is duplicate of #6252 and already fixed in #6338 (released in OTP 25.2) as pointed out by @jdamanalo
